aten/src/ATen/core/function_schema.h

```diff
@@ -72,6 +72,14 @@ struct Argument {
           name(),
           "' to be of type 'Tensor' ",
           "because it was not annotated with an explicit type.\n");
+    } else if(type()->repr_str() == "float" || type()->repr_str() == "int") {
+      if (actual_type == "int" || actual_type == "float") {
+      inferred_type_hint = c10::str(
```

Comment:
Nit:
Consider using a different variable name because technically this error message is not about "inferred type".